tests/deployment_tests/data/docker_locale/Buyer.java

```diff
@@ -18,11 +19,11 @@ public void setCash(int cash)
     this.cash=cash;
   }
 
-  public void buy(Seller seller)
```

Comment:
Please keep this as seller. The change needs to be reflected in `if(x.sell()==1)` as well.

Reply:
This file is not renamed. It is directly taken from the docs/examples directory.